src/main/java/com/spotify/docker/client/DefaultDockerClient.java

```diff
@@ -426,6 +427,18 @@ protected DefaultDockerClient(final Builder builder) {
     this.headers = new HashMap<>(builder.headers());
   }
 
+  private ClientConfig updateProxy(ClientConfig config) {
+    final String proxyHost = System.getProperty("http.proxyHost");
+    if (proxyHost != null) {
+      config.property(ClientProperties.PROXY_URI, proxyHost + ":"
+              + System.getProperty("http.proxyPort", "8080"));
```

Comment:
ah sorry I misread that this is accessing a different property. I don't think it is safe to default to 8080 though, that doesn't seem to be a good idea to me.